src/commands/Admin/purge.ts

```diff
@@ -90,26 +91,33 @@ The number represents the number of messages to purge. Maximum: ${MAX_MSGS_THAT_
     {
     */
     //Perform the deletion
-    const successful = await purge_messages(message, (numMessagesToDelete.value)); //Delete purge command as well
+    let numberOfMessagesActuallyDeleted = await purge_messages(message, (numMessagesToDelete.value)); //Delete purge command as well
```

Comment:
Why isn't this const?

Reply:
Also, why not just call it numMessagesActuallyDeleted for consistency? Don't see the need for such a long name